<launch>

  <node name="ekf_flight" pkg="state_estimation" type="ekf_node" respawn="false" output="screen" clear_params="true">
    <remap from="imu" to="/x8/output/imu" />
    <remap from="rel_pose" to="/$(arg source)/tracker_pose_cov" />
    <remap from="ekf_node/pose" to="x8/output/pose" />
    <remap from="ekf_node/velocity" to="x8/output/velocity" />
    
    <param name="sensor_offset_source" type="string" value="file" />
    <param name="relative_pose_sensor_file" type="string" value="$(find state_estimation)/calibration/vision_pose.yaml" />
    
    <param name="filter_frame" type="string" value="filter_$(arg source)" />
    <param name="imu_only_yaw" type="bool" value="true" />
  </node>

</launch>
